src/middlewared/middlewared/job.py

```diff
@@ -230,7 +230,7 @@ def __init__(self, middleware, method_name, serviceobj, method, args, options, p
         }
         self.time_started = datetime.now()
         self.time_finished = None
-        self.loop = None
+        self.loop = asyncio.get_event_loop()
```

Comment:
This won't get caught with the asyncio.CancelledError like before. Is that a problem?

Reply:
No, that exception is for when th coroutine is running

Reply:
Just curious: why?

Reply:
It will create a race condition on wait_sync() if the job has not started at the time it was called.